bedevere/news.py

```diff
@@ -29,10 +29,13 @@
 
 HELP = f"""\
 Most changes to Python [require a NEWS entry]\
-(https://devguide.python.org/committing/#updating-news-and-what-s-new-in-python).
+(https://devguide.python.org/committing/#updating-news-and-what-s-new-in-python). \
+Add one using the [blurb_it]({BLURB_IT_URL}) web app or \
+the [blurb]({BLURB_PYPI_URL}) command-line tool.
 
-Please add it using the [blurb_it]({BLURB_IT_URL}) web app or the [blurb]\
-({BLURB_PYPI_URL}) command-line tool."""
+If this change has little impact on Python users, wait for a maintainer to \
+apply the `skip news` label instead.
```

Comment:
Flip the order, as trivial changes are more likely for new contributors: 

```suggestion
If this change has little impact on Python users, wait for a maintainer to \
apply the `skip news` label.

Otherwise, [add a `NEWS` entry]\
(https://devguide.python.org/committing/#updating-news-and-what-s-new-in-python) \
using the [blurb_it]({BLURB_IT_URL}) web app or \
the [blurb]({BLURB_PYPI_URL}) command-line tool.
```

Reply:
Discussed this in person and we decided to keep things as-is for now.